### PR TITLE
feat: add show password for all passwords input

### PR DESCRIPTION
**Pull request by ykhedher**
I know that this task was not in the backlog, but I really needed this feature when I log in with a new user.

Screen Shot 2021-12-30 at 10 59 16 AM

### CODE REVIEW

**Comment by ykhedher** [comment]
[Add show password for all passwords input](https://trello.com/c/D1bvEDtY/690-add-show-password-for-all-passwords-input)

File: src/component/admin/users/change-password-component.jsx [hunk 1]
```diff
@@ -20,8 +27,17 @@ function ChangePassword({
...
     const [error, setError] = useState({});
     const [validPassword, setValidPassword] = useState(false);
+    const [showPassword, setShowPassword] = useState(false);
     const commonStyles = useCommonStyles();
 
+    const handleClickShowPassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    const handleMouseDownPassword = e => {
```
Comment (FredrikOseberg) [inline src/component/admin/users/change-password-component.jsx:37]: What is the function of this?

Comment (ykhedher) [reply src/component/admin/users/change-password-component.jsx:37]: Just following the Material UI example here: https://mui.com/components/text-fields/#input-adornments

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
I would prefer this to be encapsulated in a single PasswordField component that handles the usecase instead of repeating the functionality four times. It gives us a single point of entry for change later on.

**Comment by ykhedher** [comment]
> I would prefer this to be encapsulated in a single PasswordField component that handles the usecase instead of repeating the functionality four times. It gives us a single point of entry for change later on.

Totally agree 👍

**Review by FredrikOseberg — APPROVED** [review]
Cool. Looks good to me!

**Pull request merged**